src/components/highlight/highlight/highlight.ts

```diff
@@ -34,50 +34,24 @@ export class Highlight extends Component<Props, SpreadsheetChildEnv> {
   });
 
   onResizeHighlight(isLeft: boolean, isTop: boolean) {
-    const activeSheet = this.env.model.getters.getActiveSheet();
+    const activeSheetId = this.env.model.getters.getActiveSheetId();
 
     this.highlightState.shiftingMode = "isResizing";
     const z = this.props.zone;
 
-    const pivotCol = isLeft ? z.right : z.left;
-    const pivotRow = isTop ? z.bottom : z.top;
     let lastCol = isLeft ? z.left : z.right;
     let lastRow = isTop ? z.top : z.bottom;
     let currentZone = z;
 
     this.env.model.dispatch("START_CHANGE_HIGHLIGHT", {
-      range: this.env.model.getters.getRangeDataFromZone(activeSheet.id, currentZone),
+      range: this.env.model.getters.getRangeDataFromZone(activeSheetId, currentZone),
     });
 
-    const mouseMove = (col, row) => {
-      if (lastCol !== col || lastRow !== row) {
-        const activeSheetId = this.env.model.getters.getActiveSheetId();
-        lastCol = clip(
-          col === -1 ? lastCol : col,
-          0,
-          this.env.model.getters.getNumberCols(activeSheetId) - 1
-        );
-        lastRow = clip(
-          row === -1 ? lastRow : row,
-          0,
-          this.env.model.getters.getNumberRows(activeSheetId) - 1
-        );
-
-        let newZone: Zone = {
-          left: Math.min(pivotCol, lastCol),
-          top: Math.min(pivotRow, lastRow),
-          right: Math.max(pivotCol, lastCol),
-          bottom: Math.max(pivotRow, lastRow),
-        };
-
-        newZone = this.env.model.getters.expandZone(activeSheetId, newZone);
-
-        if (!isEqual(newZone, currentZone)) {
-          this.env.model.dispatch("CHANGE_HIGHLIGHT", {
```

Comment:
`CHANGE_HIGHLIGHT` does not seems to be used anymore and can be removed :)
(Tests need to be adapted)

Reply:
yup, I still need to adapt the tests :'( I also want to rename start_hcange_highlight to setlect_highlight, it describes it better